src/components/Article.jsx

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const Article = () => {
```

Comment:
I see you have made a single Article component but there is no link to view a single article